### ISSUE TITLE
Install errors on macOS High Sierra 

### ISSUE THREAD

**Issue by Hillsie** [BrowserSync/browser-sync#1468]
### Issue details
There are a few errors thrown when installing.

### Steps to reproduce/test case
Latest version of macOS High Sierra.
Updated node and npm and express versions
```sudo npm install -g browser-sync```

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [ 2.18.13  ]
- Node        [ v9.3.0 ]
- Npm         [5.6.0   ]

### Affected platforms

- [ ] linux
- [ ] windows
- [X] OS X
- [ ] freebsd
- [ ] solaris
- [ ] other _(please specify which)_

### Browsersync use-case

- [ ] API
- [ ] Gulp
- [ ] Grunt
- [X ] CLI

### If CLI, please paste the entire command below

```
sudo npm install -g browser-sync
npm WARN deprecated express@2.5.11: express 2.x series is deprecated
npm WARN deprecated connect@1.9.2: connect 1.x series is deprecated
/usr/local/bin/browser-sync -> /usr/local/lib/node_modules/browser-sync/bin/browser-sync.js

> fsevents@1.1.3 install /usr/local/lib/node_modules/browser-sync/node_modules/fsevents
> node install

node-pre-gyp ERR! Tried to download(undefined): https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.1.3/fse-v1.1.3-node-v59-darwin-x64.tar.gz
node-pre-gyp ERR! Pre-built binaries not found for fsevents@1.1.3 and node@9.3.0 (node-v59 ABI, unknown) (falling back to source compile with node-gyp)
gyp ERR! clean error
gyp ERR! stack Error: EACCES: permission denied, rmdir 'build'
gyp ERR! System Darwin 17.3.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "clean"
gyp ERR! cwd /usr/local/lib/node_modules/browser-sync/node_modules/fsevents
gyp ERR! node -v v9.3.0
gyp ERR! node-gyp -v v3.6.2
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js clean' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/browser-sync/node_modules/fsevents/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:159:13)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:943:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:220:5)
node-pre-gyp ERR! System Darwin 17.3.0
node-pre-gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/browser-sync/node_modules/fsevents/node_modules/node-pre-gyp/bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /usr/local/lib/node_modules/browser-sync/node_modules/fsevents
node-pre-gyp ERR! node -v v9.3.0
node-pre-gyp ERR! node-pre-gyp -v v0.6.39
node-pre-gyp ERR! not ok
Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js clean' (1)
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.3 (node_modules/browser-sync/node_modules/fsevents):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.3 install: `node install`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1
```

**Comment by IkwhanChang** [comment]
It seems like fsevent's error. (see: https://github.com/strongloop/fsevents/issues/194)
I think fsevent still need to fix error related Node 9.3.0. It has an ongoing branch (https://github.com/strongloop/fsevents/tree/fix-93) but it still working on it. But when I upgrade the same version of yours (node 9.3.0 npm 5.6.0 with Mac OS High Sierra v.10.13.2) I simply remove 'sudo' and run the command of 'npm install -g browser-sync' then it seems like working.

Below is my CLI message:
$ npm install -g browser-sync
npm WARN deprecated express@2.5.11: express 2.x series is deprecated
npm WARN deprecated connect@1.9.2: connect 1.x series is deprecated
/usr/local/bin/browser-sync -> /usr/local/lib/node_modules/browser-sync/bin/browser-sync.js

> fsevents@1.1.3 install /usr/local/lib/node_modules/browser-sync/node_modules/fsevents
> node install

[fsevents] Success: "/usr/local/lib/node_modules/browser-sync/node_modules/fsevents/lib/binding/Release/node-v59-darwin-x64/fse.node" is installed via remote
+ browser-sync@2.18.13
added 413 packages in 7.632s

Hope it works for you.

**Comment by Hillsie** [comment]
YIp, I think it's a node issue as you point out. It worked anyway. Probably something to do with the fact that I don't have. Looks like it was optional.
```gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp```

Thanks, I'll close it off.
